text-generation-inference/Dockerfile

```diff
@@ -106,10 +107,9 @@ ARG ACCELERATE_VERSION='0.27.2'
 ARG SAFETENSORS_VERSION='0.4.2'
 
 # Install Torch and XLA packages for TPU
-# RUN pip install "torch~=2.2.0" "torch_xla[tpu]~=2.2.0" -f https://storage.googleapis.com/libtpu-releases/index.html
-RUN python3 -m pip install hf_transfer safetensors==${SAFETENSORS_VERSION} accelerate==${ACCELERATE_VERSION} && \
-    cd /opt/optimum-tpu && \
-    python3 -m pip install .
+RUN python3 -m pip install "torch~=2.2.0" "torch_xla[tpu]~=2.2.0" -f https://storage.googleapis.com/libtpu-releases/index.html && \
+    python3 -m pip install hf_transfer safetensors==${SAFETENSORS_VERSION} accelerate==${ACCELERATE_VERSION} && \
+    SETUPTOOLS_SCM_PRETEND_VERSION_FOR_OPTIMUM_TPU=${VERSION} python3 -m pip install -e /opt/optimum-tpu
```

Comment:
what is SETUPTOOLS_SCM_PRETEND_VERSION_FOR_OPTIMUM_TPU ?

Reply:
@ManfeiBai